docs/sources/developers/plugins/_index.md

```diff
@@ -24,8 +24,8 @@ Follow the questions and you will have a starter plugin ready to develop.
 
 If you want a more guided introduction to plugin development, check out our tutorials:
 
-- [Build a panel plugin](/tutorials/build-a-panel-plugin/)
-- [Build a data source plugin](/tutorials/build-a-data-source-plugin/)
+- [Build a panel plugin](../../tutorials/build-a-panel-plugin/)
+- [Build a data source plugin](../../tutorials/build-a-data-source-plugin/)
```

Comment:
This is weird, I see that a link like this one is working on a different page: 

https://github.com/grafana/grafana/blob/main/docs/sources/administration/data-source-management/index.md?plain=1#L207

https://grafana.com/docs/grafana/latest/administration/data-source-management/#create-a-data-source-plugin

But it's not working in the rest. I have seen more examples of this. You may want to search for all the appearances of `/tutorials/` in this folder, like:

https://github.com/grafana/grafana/blob/main/docs/sources/developers/plugins/add-support-for-annotations.md?plain=1#L9

I also see that in other places, the whole URL is set, which seems more robust to me and works both in Github preview mode and our docs site. That's probably a better option unless we have a reason for not doing that:

https://github.com/grafana/grafana/blob/main/docs/sources/developers/angular_deprecation.md?plain=1#L37

Reply:
> I also see that in other places, the whole URL is set, which seems more robust to me and works both in Github preview mode and our docs site. That's probably a better option unless we have a reason for not doing that:

At the moment, maintaining correct links on the GitHub preview is not a priority but I would like to support it in the future so I have opened https://github.com/grafana/website/issues/12070 with the website team for how we can do so.

The reason for using [partial URIs](https://www.rfc-editor.org/rfc/rfc1630) is that the website is served from different hostnames, grafana.com is the public website, but there are also staging sites with different hostnames.

Reply:
I think the reason for some of the confusion is that these links are only broken in latest where these links are different -- https://github.com/grafana/grafana/blob/v9.4.x/docs/sources/developers/plugins/_index.md?plain=1#L27-L44.

This is my fault because I failed to backport https://github.com/grafana/grafana/commit/66b54e2dbd10fd399e2034b3367bbb579b8a0635 to this branch. I will open up the PR now to fix it.

Reply:
https://github.com/grafana/grafana/pull/64114

Reply:
got it, so these changes are not needed, right?

Reply:
I don't believe the changes in this file are needed. 

There are additional changes in the other file that might still be useful.

Reply:
> I don't believe the changes in this file are not needed.

@jdbaldry That's a double negative. To be clear, did you mean that that this PR isn't needed?

Reply:
Sorry, that's me misreading myself and editing it wrongly :D 

I believe the changes in this file are *not* needed.